detox/test/e2e/p-datePicker.js

```diff
@@ -0,0 +1,17 @@
+describe('DatePicker', () => {
+    beforeEach(async () => {
+      await device.reloadReactNative();
+    });
+  
+    beforeEach(async () => {
+      await element(by.text('DatePicker')).tap();
+    });
+    
+    it('check and scroll datePicker', async () => {
+        await expect(element(by.type('UIPickerView'))).toBeVisible();
+        await element(by.type('UIPickerView')).setColumnToValue(1,"6");
+        await element(by.type('UIPickerView')).setColumnToValue(2,"34");
```

Comment:
Right now, this test checks nothing. There should be a label in the screen, and on date picker change value, it should update. You should then assert that the label has the correct value.

Reply:
Checking label with text will be always true, because this label always on date picker, I think right solution is checking property of datePicker current value or something like that, now I work in this direction, add this functionality asap

Reply:
I don't understand. The label is updated with the value set in the picker. So if you select 6:41, it won't say "6:34" in the label. If you assert that `label.text == "6:34"`, you will test that the picker really changed value. That's what we want to test in the testing suite.

Reply:
In my example if element (UIPickerView) can't find variable with current column or value test will be failed

Reply:
But that is not what we test in the suite. We test that the native system behaves as expected following Detox changes.